code/modules/mob/living/carbon/human/species_types/psyphoza.dm

```diff
@@ -0,0 +1,419 @@
+
+	mutant_bodyparts = list("psyphoza_cap")
+	default_features = list("psyphoza_cap" = "Portobello", "body_size" = "Normal", "mcolor" = "fff")
+	hair_color = "fixedmutcolor"
+
+	species_chest = /obj/item/bodypart/chest/psyphoza
+	species_head = /obj/item/bodypart/head/psyphoza
+	species_l_arm = /obj/item/bodypart/l_arm/psyphoza
+	species_r_arm = /obj/item/bodypart/r_arm/psyphoza
+	species_l_leg = /obj/item/bodypart/l_leg/psyphoza
+	species_r_leg = /obj/item/bodypart/r_leg/psyphoza
+
+	//Fire bad!
+	burnmod = PSYPHOZA_BURNMOD
+
+	//Reference to psychic highlight action
+	var/datum/action/item_action/organ_action/psychic_highlight/PH
+
+/datum/species/psyphoza/on_species_gain(mob/living/carbon/C, datum/species/old_species, pref_load)
+	. = ..()
+	PH = locate(/datum/action/item_action/organ_action/psychic_highlight) in C.actions
+
+/datum/species/psyphoza/on_species_loss(mob/living/carbon/human/C, datum/species/new_species, pref_load)
+	. = ..()
+	PH = null
+
+/datum/species/psyphoza/random_name(gender, unique, lastname, attempts)
+	. = "[pick(GLOB.psyphoza_first_names)] [pick(GLOB.psyphoza_last_names)]"
+	if(unique && attempts < 10 && findname(.))
+		return .(gender, TRUE, null, ++attempts)
+
+/datum/species/psyphoza/handle_chemicals(datum/reagent/chem, mob/living/carbon/human/H)
+	if(istype(chem, /datum/reagent/drug) && H.blood_volume < BLOOD_VOLUME_NORMAL)
+		H.blood_volume += chem.volume * 15
+		H.reagents.remove_reagent(chem.type, chem.volume)
+		return FALSE
+	return ..()
+
+/datum/species/psyphoza/get_scream_sound(mob/living/carbon/user)
+	return pick('sound/voice/psyphoza/psyphoza_scream_1.ogg', 'sound/voice/psyphoza/psyphoza_scream_2.ogg')
+
+/datum/species/psyphoza/primary_species_action()
+	. = ..()
+	PH?.Trigger()
+
+/datum/species/psyphoza/get_species_description()
+	return "..."
+
+/datum/species/psyphoza/get_species_lore()
+	return list("...")
+
+/datum/species/psyphoza/create_pref_unique_perks()
+	var/list/to_add = list()
+
+	to_add += list(
+		list(
+			SPECIES_PERK_TYPE = SPECIES_POSITIVE_PERK,
+			SPECIES_PERK_ICON = "lightbulb",
+			SPECIES_PERK_NAME = "Psychic",
+			SPECIES_PERK_DESC = "Psyphoza are psychic and can sense things others can't.",
+		),
+		list(
+			SPECIES_PERK_TYPE = SPECIES_NEGATIVE_PERK,
+			SPECIES_PERK_ICON = "eye",
+			SPECIES_PERK_NAME = "Blind",
+			SPECIES_PERK_DESC = "Psyphoza are blind and can't see outside their immediate location and psychic sense.",
+		),
+	)
+
+	return to_add
+
+//This originally held the psychic action until I moved it to the eyes, keep it please.
+/obj/item/organ/brain/psyphoza
+	name = "psyphoza brain"
+	desc = "Bubbling with psychic energy...no wait...that's blood."
+	color = "#ff00ee"
+
+// PSYCHIC ECHOLOCATION
+/datum/action/item_action/organ_action/psychic_highlight
+	name = "Psychic Sense"
+	icon_icon = 'icons/mob/actions.dmi'
+	button_icon_state = "activate_psychic"
+	transparent_when_unavailable = TRUE
+	///The distant our psychic sense works
+	var/psychic_scale = 2.28
```

Comment:
You ate the description in the last commit

Reply:
Whoops. 
Thanks, Penwin.